src/bio/bio_xstream.c

```diff
@@ -1716,6 +1719,41 @@ scan_bio_bdevs(struct bio_xs_context *ctxt, uint64_t now)
 	nvme_glb.bd_scan_age = now;
 }
 
+void
+bio_led_event_monitor(struct bio_xs_context *ctxt, uint64_t now)
+{
+	struct bio_bdev         *d_bdev;
+	struct spdk_bdev        *bdev;
+	static uint64_t          led_event_period = NVME_LED_EVENT_PERIOD;
+
+	for (bdev = spdk_bdev_first(); bdev != NULL;
+	     bdev = spdk_bdev_next(bdev)) {
```

Comment:
Why not iterate bio_bdev list directly?

Reply:
is it necessary to do this?

Reply:
I mean here you just need to iterate the bio_bdev list directly.  Instead of iterating SPDK bdev, then find corresponding bio_bdev by device name.

Reply:
will update